source/Kepler-System/Kepler.class.st

```diff
@@ -0,0 +1,28 @@
+"
+I'm a Shared Pool providing access to the Kepler namespace
```

Comment:
Why is this interface registration in a SharedPool instead of a manifest like the other uses? Which is correct?

Reply:
Because in this package we already have a good place to put it :smile: . We can make it consistent, but in this case I will prefer to let it here and not create the manifest class just for that.